data/core.yaml

```diff
@@ -316,6 +316,7 @@ en:
       relation: These features can't be merged because they have conflicting relation roles.
```

Comment:
This string could be renamed to `conflicting_relation_roles` since there’s now another one about overall relation membership.

Reply:
I don't really want to touch connect.js in this PR.